### PR TITLE
fix(local-debug): show hint to FAQ page when local debug sideloading

### PR DESCRIPTION
**Pull request by kuojianlu**
- update local debug help doc
- show information message guiding user to open FAQ page when local debug sideloading
- FAQ telemetry

image

### CODE REVIEW

File: packages/vscode-extension/src/debug/constants.ts [hunk 1]
```diff
@@ -86,3 +86,11 @@ Add any other context about the problem here.
 export const errorDetail = `
 **Error detail**
 `;
+
+export enum SideloadingHintStateKeys {
+  DoNotShowAgain = "sideloadingHint/doNotShowAgain",
+}
+export const sideloadingHintMessage =
+  "If any error occurs during local debug, you can refer to the FAQ page by clicking 'Open FAQ' button.";
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/debug/constants.ts:94]: Why not move to Strings.json?

**Review by swatDong — APPROVED** [review]

File: packages/vscode-extension/src/debug/constants.ts (hunk 1 above)
Comment (kuojianlu) [reply packages/vscode-extension/src/debug/constants.ts]: Nice idea. Moved to `Strings.json` as well as `npmInstallFailedHintMessage` before. Thanks~

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Comment by kuojianlu** [comment]
@chagong Please help review the new telemetry event and property. Thanks.

**Review by chagong — APPROVED** [review]

**Pull request merged**